spec/solidus_content/content_providers/solidus_static_content_spec.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe SolidusContent::ContentProviders::SolidusStaticContent do
+  let(:page) { create(:page) }
+
+  context 'passing the slug in the options' do
+    it 'returns static content data' do
+      expect(
+        described_class.call(slug: page.slug)[:data]
+      ).to include(page.attributes.symbolize_keys)
```

Comment:
Reloading the record before comparing it might fix the error

Reply:
Thank you, it works